docs/mid.md

```diff
@@ -0,0 +1,131 @@
+---
+id: mid
+title: Mid
+---
+
+## Installation
+`"ru.tinkoff" %% "tofu" % tofu-version`  
+or as a standalone dependency:   
+`"ru.tinkoff" %% "tofu-core-higher-kind" % tofu-version` 
+
+## Assumption
+
+Consider some trait
+```scala
+trait MyBusinessModule[F[_]]{
+  def doBusinessThing(entity: Entity, info: Info): F[Value]
+  def undoBusinessThing(entity: Entity): F[Respect]
+}
+```
+Often `F` presented like some `IO` or any transformer
```

Comment:
Shouldn't we add the fact that `F` could some kind of reader too? I feel like this is not being told enough 😞

Reply:
Added